src/react/utils/utils.js

```diff
@@ -200,3 +202,35 @@ export const getScrollToId = (entries, key) => {
 
   return `id_${entries[0].id}`;
 };
+
+/**
+ * Sanitize HTML for output. Help prevent XSS attacks.
+ *
+ * @link https://www.npmjs.com/package/dompurify
+ * @param  {string} dirty HTML to sanitize
+ * @return {string} sanitized HTML
+ */
+export const sanitizeHTML = (dirty) => {
+  // Whitelist iframes for the plugins 'embded media' feature.
+  const iframeWhitelist = [
+    'www.hulu.com',
+    'player.hulu.com',
+    'open.spotify.com',
+    'player.vimeo.com',
+    'www.youtube.com',
+    // Instagram and Twitter don't use iframes.
```

Comment:
Does this mean we will strip out Twitter and Instagram embeds?

Reply:
Also, how was this list of domains generated?

Reply:
This list of domains was generated taking liveblogs advertised media embed features, adding them to an entry, and seeing what WordPress/oembed gave back. In writing that, I realize I need to take a closer look into these domains (ie: subdomain changes for different regions or languages) and each services documentation to ensure its a single primary domain for everything.

& as pointed out by @sboisvert, a filter to manage this whitelist would be needed.